src/bot/bot.py

```
@@ -1644,5 +1775,11 @@ async def help(interaction: discord.Interaction):
     try:
         bot_logger.info("Starting bot...")
         bot.run(TOKEN)
-    except Exception as e:
-        bot_logger.critical(f'Failed to start bot: {e}', exc_info=True)
+    except discord.errors.LoginFailure as bot_login_error:
+        bot_logger.critical("Login failed: %s", bot_login_error)
+    except discord.errors.ConnectionClosed as bot_conn_error:
+        bot_logger.critical("Connection closed: %s", bot_conn_error)
+    except ConnectionError as bot_net_error:
+        bot_logger.critical("Network error: %s", bot_net_error)
+    except (discord.DiscordException, ConnectionError) as bot_error:
```

Ruff (0.8.2)

try-except block with duplicate exception `ConnectionError`

(B025)